### PR TITLE
Issue2302 building time series enhancement

### PR DESCRIPTION
**Pull request by dhblum**
This is to close https://github.com/lbl-srg/modelica-buildings/issues/2302.  Still to be done:

- [x] Update release notes

### CODE REVIEW

**Comment by AntoineGautier** [comment]
I realize we got confused about what `facMul` stands for in `Buildings.Experimental.DHC.Loads.BaseClasses.Examples.BaseClasses.BuildingTimeSeries`. 
Also, `facMulZon` is not to be used when providing loads as time series. 
In the commit a6188a6

- I changed some parameter names to make them more explicit, 
- I fixed the default parameter bindings,
- I updated the documentation of (formerly) example models that were (quickly) promoted as generic models,
- I did NOT update the reference results so that you can check the (minor) impact of those modifications.

Perhaps the best is to discuss those changes together during our next meeting.

**Comment by AntoineGautier** [comment]
@dhblum Please see my last commits for the update of the documentation and release notes. I believe the reference results still need to be updated as well.

**Comment by dhblum** [comment]
Thanks @AntoineGautier I think your updates look good.  I updated unit test results and will see if tests pass.

**Review by AntoineGautier — APPROVED** [review]
This is ready to merge from my end.

**Comment by dhblum** [comment]
Thanks @AntoineGautier.  This is not yet ready to merge.  @mwetter made some comments in a discussion last month with regards to clarifying the parameterization of ``facMulHea/Coo`` by adding intermediate parameter(s) that are easier to interpret and making ``facMulHea/Coo`` a ratio of those parameter(s).

File: Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo [hunk 1]
```diff
@@ -18,12 +18,16 @@ model BuildingTimeSeries
...
-  parameter Real facMulHea=1
+  parameter Real facMulHea=QHea_flow_nominal /
+    (5e3 * abs(T_aLoaHea_nominal - T_aHeaWat_nominal) / 20 *
+     mLoaHea_flow_nominal / 1)
     "Heating terminal unit multiplier factor"
-    annotation(Dialog(enable=have_heaWat, group="Scaling"));
-  parameter Real facMulCoo=1
+    annotation(Dialog(enable=have_heaWat, group="Scaling", tab="Advanced"));
+  parameter Real facMulCoo=QCoo_flow_nominal /
+    (-3e3 * abs(T_aLoaCoo_nominal - T_aChiWat_nominal) / 15 *
```
Comment (mwetter) [inline Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo:27]: This is a similar problem as reported in https://github.com/lbl-srg/modelica-buildings/pull/3093#issuecomment-1384480665
These factors should be removed or explained in the `info` section and introduced as parameters so we know what they represent.

**Review by mwetter — CHANGES_REQUESTED** [review]
See inline comment regarding the literal values applied to the multiplier.

File: Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo (hunk 1 above)
Comment (dhblum) [reply Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo:27]: I am working on edits also with input from @AntoineGautier.

**Comment by dhblum** [comment]
@AntoineGautier Can you let me know what you think of changes in https://github.com/lbl-srg/modelica-buildings/pull/3165/commits/2a947a50f73eec6481aab9d9ae0e241cded5ffee?  It makes the reference terminal unit performance conditions parameters in the "Advanced" tab and edits the documentation a bit.

**Comment by AntoineGautier** [comment]
I double checked the manufacturer data sheet and realized that there was an error in unit conversion from their documentation, see below. 
I updated the selection conditions accordingly and rather referenced the "Advanced" tab in the documentation, as opposed to providing numerical values.
I also modified the way the cooling unit multiplier factor is computed to better match the eps-NTU formulation in wet regime.

Capture d’écran 2023-01-18 110632

**Comment by dhblum** [comment]
Thanks @AntoineGautier.  I've reviewed your changes and they look good to me.  I've updated the reference results, which have expected and reasonable changes.  In the 5G Reservoir Network models (Buildings.Experimental.DHC.Examples.Combined.SeriesConstantFlow/SeriesVariableFlow), the changes in ETS pump energy by the end of the test period are less than 0.5%.

File: Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo (hunk 1 above)
Comment (dhblum) [reply Buildings/Experimental/DHC/Loads/BaseClasses/Examples/BaseClasses/BuildingTimeSeries.mo]: @mwetter Can you check again and see if the latest changes are satisfactory?

**Comment by dhblum** [comment]
Still to-do:

- [ ] Update release notes

**Review by mwetter — APPROVED** [review]

**Pull request merged**